Wikipedia/Code/WebViewController.m

```diff
@@ -687,8 +687,11 @@ - (void)updateReferencesHeightAndBottomConstraints {
 }
 
 - (CGFloat)getRefsPanelHeight {
-    // TODO: this code could be changed to use size classes instead of orientation which is deprecated from iOS 8
+    WMF_TECH_DEBT_WARN(use size classes instead of interfaceOrientation)
+#pragma clang diagnostic push
```

Comment:
@montehurd remember, with great power, comes great responsibility.

![oh ben...](http://i.imgur.com/TnsFL5S.gif)

Specifically, I'm using it here because I'm specifically :football: punting fixing this until later—but the rest of the app is clear of deprecation warnings. So, I'm special-casing this LOC so the rest of the app can be protected against deprecated method usage.